concepts/05 UI Components/TreeList/10 Columns/50 Adaptability/05 Hide Columns.md

```diff
@@ -0,0 +1,97 @@
+To enable column hiding, set the [columnHidingEnabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#columnHidingEnabled) property to **true**.
+
+Each column has a unique default hiding priority. The rightmost column has the priority of 0. This value is incremented by 1 for columns from right to left; column with the lowest priority is hidden first.
+
+You can use the **columns[]**.[hidingPriority](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#hidingPriority) property to specify custom hiding priorities only for those columns that you want to hide. This will override the default behavior described above. In this case, the **columnHidingEnabled** property can be omitted.
+
+[note]
+
+If a column's cell truncates text due to its length, TreeList can hide columns until the full text is displayed. To enable this, ensure that the column's [width](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#width) is not set or set as a percentage and configure column hiding. You can also enable this feature to prevent horizontal scrolling.
+
+[/note]
```

Comment:
I think that this information is unnecessary, but let's wait until we figure out why developers wanted to add it in the first place.

```suggestion
```